app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -282,6 +289,16 @@ public boolean onItemLongClick(AdapterView<?> parent, View v, int pos, long id)
         applyDesignTweaks();
     }
 
+    private void adjustInputType(String currentText) {
+        if (currentText != null && Pattern.matches("[+]\\d+", currentText)) {
+            searchEditText.setInputType(InputType.TYPE_CLASS_PHONE);
+        } else if (searchEditTextSpellcheck) {
+            searchEditText.setInputType(SPELLCHECK_ENABLED_TYPE);
+        } else {
+            searchEditText.setInputType(InputType.TYPE_CLASS_TEXT);
```

Comment:
Sadly, this should instead be `textVisiblePassword|textNoSuggestions` otherwise some keyboards will still display suggestions.

See for instance https://github.com/Neamar/KISS/issues/44 :(